startup.sh

```diff
@@ -3,10 +3,10 @@
 bundle install
 
 if [[ $1 == "--live" ]] ; then
-  GOVUK_APP_DOMAIN=www.gov.uk \
-  GOVUK_WEBSITE_ROOT=https://www.gov.uk \
-  PLEK_SERVICE_SEARCH_URI=${PLEK_SERVICE_SEARCH_URI-https://www.gov.uk/api} \
-  PLEK_SERVICE_CONTENT_STORE_URI=${PLEK_SERVICE_CONTENT_STORE_URI-https://www.gov.uk/api} \
+  GOVUK_APP_DOMAIN=www-origin.integration.publishing.service.gov.uk \
```

Comment:
Did you mean to commit this?

Reply:
:face_with_head_bandage: Nope, rebased over this.